libvirt.go

```diff
@@ -138,9 +139,7 @@ func (l *Libvirt) authenticate() error {
 	return nil
 }
 
-// ConnectToURI establishes communication with the specified libvirt driver
-// The underlying libvirt socket connection must be previously established.
-func (l *Libvirt) ConnectToURI(uri ConnectURI) error {
+func (l *Libvirt) initLibvirtComms(uri ConnectURI) error {
```

Comment:
This is still using the manual payload construction from before the code generator. It should really call the generated ProcConnectOpen instead.

Reply:
I'd rather not put that in the same PR as a bunch of code re-org, but I might be willing to take bribes to put it into a follow-up PR.